modules/web/src/app/node-data/basic/provider/kubevirt/topology-spread-contraint-form/template.html

```diff
@@ -37,7 +37,8 @@
         <input matInput
                [name]="Controls.TopologyKey"
                [formControlName]="Controls.TopologyKey"
-               (keyup)="onControlValueChange(constraint)">
+               (keyup)="onControlValueChange(constraint)"
+               valueChanged>
```

Comment:
This component is similar to label-form and taint-form so it should be updated in similar manner.